schema/resolvers.ts

```diff
@@ -78,6 +78,23 @@ export const resolvers = {
         .populate('portfolio');
       return response;
     },
+
+    async application(
+      parent: any,
+      { _id }: any
+    ): Promise<ApplicationInterface> {
+      const response = await application.findById(_id).populate('referred_by');
+      return response;
+    },
+
+    async consultation(
+      parent: any,
+      { _id }: any
+    ): Promise<ConsultationInterface> {
+      const response = await consultation.findById(_id).populate('referred_by');
```

Comment:
for the populate on `referred_by`, I think the consultation schema doesn't have such field in it.